profile/README.md

```diff
@@ -18,7 +18,7 @@
 |---|---| 
 | [imap_processing](https://github.com/IMAP-Science-Operations-Center/imap_processing) | Software for performing L0->L3 data processing | 
 | [sds-data-manager](https://github.com/IMAP-Science-Operations-Center/sds-data-manager) | Software for building and configuring AWS architecture to support data processing | 
-| [sds-access-lib](https://github.com/IMAP-Science-Operations-Center/sds-access-lib) | Contains a simple script that allows a user to upload, query, and download data from the IMAP Science Data Center | 
+| [imap-data-access](https://github.com/IMAP-Science-Operations-Center/imap-data-access) | Tools for users to upload, download, and query data from the IMAP Science Data Center | 
```

Comment:
Do we want to alphabetize `download, query, upload`?

Reply:
Maybe "Package and command line utility for users to ..."?
Just wordsmithing, feel free to ignore!